### ISSUE TITLE
Using Group attention get worse result

### ISSUE THREAD

**Issue by zhenghao977** [carrierlxk/COSNet#42]
Hello, I use the siamese_model_conf_try_single.py and train_iteration_conf_group.py to train a model with two reference images, and I can only get 61.1 J-Mean. I don't modify the parameters during training. and I think siamese_model_conf_try_single.py :
 A = F.softmax(A, dim = 1) #           
 B = F.softmax(torch.transpose(A,1,2),dim=1)
 maybe:
 A1 = F.softmax(A, dim = 1) #           
 B = F.softmax(torch.transpose(A,1,2),dim=1)
